casper/src/main/scala/coop/rchain/casper/util/comm/DeployService.scala

```diff
@@ -17,6 +15,8 @@ trait DeployService[F[_]] {
   def showBlock(q: BlockQuery): F[String]
   def showBlocks(): F[String]
   def addBlock(b: BlockMessage): F[(Boolean, String)]
+  def listenForDataAtName(request: Channel): F[ListeningNameDataResponse]
+  def listenForContinuationAtName(request: Channels): F[ListeningNameContinuationResponse]
```

Comment:
Why does `listenForContinuationAtName` takes as an argument `Channels` (`Seq[Channel]`) if you always wrap single `Channel` with `Seq` ? Can't it be a `Channel` ?